src/RestApi/StoreApi/Schemas/CartShippingRateSchema.php

```diff
@@ -0,0 +1,242 @@
+<?php
+/**
+ * Cart shipping rate schema.
+ *
+ * @package WooCommerce/Blocks
+ */
+
+namespace Automattic\WooCommerce\Blocks\RestApi\StoreApi\Schemas;
+
+defined( 'ABSPATH' ) || exit;
+
+use \WC_Shipping_Rate as ShippingRate;
+
+/**
+ * CartShippingRateSchema class.
+ */
+class CartShippingRateSchema extends AbstractSchema {
+	/**
+	 * The schema item name.
+	 *
+	 * @var string
+	 */
+	protected $title = 'cart-shipping-rate';
+
+	/**
+	 * Cart schema properties.
+	 *
+	 * @return array
+	 */
+	protected function get_properties() {
+		return [
+			'destination'    => [
+				'description' => __( 'Shipping destination address.', 'woo-gutenberg-products-block' ),
+				'type'        => 'object',
+				'context'     => [ 'view', 'edit' ],
+				'readonly'    => true,
+				'items'       => [
```

Comment:
shoudl this be `properties` not `items` (since it's an object type)

Reply:
yes and it seems there are a few like this - ill correct them here.